### ISSUE TITLE
SDK Information

### ISSUE THREAD

**Issue by HazAT** [getsentry/sentry-dotnet#100]
All SDKs should send SDK information like this:

for `Sentry` it should look like this:

```json
{
  "name": "sentry.dotnet",
  "version": "1.0.0-rc2",
  "packages": [
    {
      "name": "nuget:Sentry",
      "version": "1.0.0-rc2"
    },
  ]
}
```

for `Sentry.AspNetCore` it should look like this:

```json
{
  "name": "sentry.dotnet.aspnetcore",
  "version": "1.0.0-rc2",
  "packages": [
    {
      "name": "nuget:Sentry",
      "version": "1.0.0-rc2"
    },
    {
      "name": "nuget:Sentry.AspNetCore",
      "version": "1.0.0-rc2"
    },
  ]
}
```

and so on ...

**Comment by HazAT** [comment]
@bruno-garcia almost, it's like this https://github.com/getsentry/sentry-docs/pull/293/commits/02e4ff713ca7b379645d4cfd63ba017b82ad501c

Since I think that `Sentry.AspNetCore` and `Sentry.Extensions.Logging` are both "SDKs" on their own, they deserve to bin in the namespace like:

`sentry.dotnet.aspnetcore`
and
`sentry.dotnet.extensions-logging`

does this make sense?

That way we can boil it down to the individual SDK and still filter for everything `dotnet` related.
The packages can stay the way they are.

**Comment by HazAT** [comment]
The identifiers should basically match the folders here:
https://github.com/getsentry/sentry-release-registry/tree/master/sdks

**Comment by bruno-garcia** [comment]
Closed by #105
